nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/pom.xml

```diff
@@ -48,6 +50,16 @@ language governing permissions and limitations under the License. -->
             <version>1.15.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>${common-text.version}</version>
+        </dependency>
```

Comment:
I don't see where this is used. Is this added `commons-text` dependency still needed?